### PR TITLE
Add UCT "Troubleshooting" section

### PR DESCRIPTION
**Pull request by Dnd-Mafer**
## Purpose of this pull request

This pull request (PR) add a troubleshooting on UCT usage if nothing appears in terminal.

## Affected DevDocs pages

https://devdocs.magento.com/upgrade-compatibility-tool/run.html

whatsnew
Added a 'Troubleshooting section' to the [Upgrade Compatibility Tool](https://devdocs.magento.com/upgrade-compatibility-tool/run.html#troubleshooting) topic.

### CODE REVIEW

**Comment by dobooth** [comment]
Thanks for the PR, @Dnd-Mafer. Looks like you have a trailing space on around line 150.

File: src/upgrade-compatibility-tool/run.md [hunk 1]
```diff
@@ -134,9 +134,37 @@ Regarding the GraphQL schema compatibility comparison, the output would be very
...
+{:.bs-callout-info}
+The `M2_VERSION` is the target Magento 2 version you want to compare with your own Magento with.
+
+After running this command
+
+```bash
+bin/uct upgrade:check INSTALLATION_DIR -c M2_VERSION
+```
+
+if the only output is `Upgrade compatibility tool`
```
Comment (BarnyShergold) [inline src/upgrade-compatibility-tool/run.md:157]: Should this not say "run this command:" at end of this sentence?
```suggestion
If the only output is `Upgrade compatibility tool`
```

File: src/upgrade-compatibility-tool/run.md [hunk 2]
```diff
@@ -134,9 +134,37 @@ Regarding the GraphQL schema compatibility comparison, the output would be very
...
+
+```terminal
+$ bin/uct upgrade:check /var/www/project/magento/ -c 2.4.1
```
Comment (BarnyShergold) [inline src/upgrade-compatibility-tool/run.md:160]:
```suggestion
bin/uct upgrade:check /var/www/project/magento/ -c 2.4.1
```

File: src/upgrade-compatibility-tool/run.md [hunk 3]
```diff
@@ -134,9 +134,37 @@ Regarding the GraphQL schema compatibility comparison, the output would be very
...
+Upgrade compatibility tool
+$
```
Comment (BarnyShergold) [inline src/upgrade-compatibility-tool/run.md:162]: Remove this line

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few suggestions

File: src/upgrade-compatibility-tool/run.md (hunk 3 above)
Comment (Dnd-Mafer) [reply src/upgrade-compatibility-tool/run.md]: Done

**Review by BarnyShergold — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8804`.

**Comment by dshevtsov** [comment]
@Dnd-Mafer, we cannot process pull requests open from the master branch. Please follow [contribution guidelines](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md#create-a-branch) and create a new pull request with your changes. Link this pull request in the description, so that an assignee and reviewers would consider the work they've done already.

**Pull request closed without merging**